pulse/broker/src/main/java/gov/ca/emsa/pulse/service/AlternateCareFacilityService.java

```diff
@@ -78,19 +92,21 @@ public AlternateCareFacility create(@RequestBody(required=true) AlternateCareFac
 
 	@ApiOperation(value = "Edit an existing ACF")
 	@RequestMapping(value = "/{acfId}/edit", method = RequestMethod.POST)
+	@PreAuthorize("hasPermissionForAcf(#acfId)")
```

Comment:
Will ROLE_PROVIDER return true for this check? Though they can create ACFs they can't edit them; this might also need a "has role: ROLE_ADMIN or ROLE_ORG_ADMIN"